docs/source/user-guide/python.md

```diff
@@ -103,14 +109,15 @@ The `explain` method can be used to show the logical and physical query plans fo
 The following example demonstrates creating arrays with PyArrow and then creating a Ballista DataFrame.
 
 ```python
-import ballista
+from ballista import Ballista
 import pyarrow
 
 # an alias
+# TODO implement Functions
 f = ballista.functions
 
 # create a context
-ctx = ballista.BallistaContext("localhost", 50050)
+ctx = StandaloneBallista.build()
```

Comment:
should this be `Ballista.standalone()` ?

Reply:
fixed

Reply:
this still shows in diff as  `StandaloneBallista.build()`

Reply:
yep, came back to this and forgot to commit the changes.